index.html

```diff
@@ -264,6 +572,8 @@ <h2>Data Model</h2>
       <li>In addition to an <a>IRI</a> or <a>blank node</a>, a <a>node</a> having an outgoing edge MAY be labeled with an <a>embedded node</a>.</li>
     </ul>
 
+    <p class="ednote">Is it reasonable to say <em>labeled</em> here? What might be better terminology?</p>
+
```

Comment:
No, I think "be labeled with" should be replace with "be". This is aligned with how the data model is described otherwise ("Every node is an IRI, a blank node or a literal").

Furthermore, I think we should remove the 2nd item of the list above, which describes "node objects" rather than nodes, i.e. the syntax rather than the data model.

Reply:
See update in 28ae5cd7.